db/schema.rb

```diff
@@ -299,6 +299,15 @@
     t.index ["student_id"], name: "index_student_assessments_on_student_id"
   end
 
+  create_table "student_photos", force: :cascade do |t|
+    t.string "student_id"
```

Comment:
Should be an integer, not a string. Also this needs an index since it's a foreign key.